packages/kit/src/packaging/test/fixtures/javascript/src/lib/internal/Test.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
-    /**
-     * @type {import('./foo').Foo}
-     */
-    export let foo;
+	/**
+	 * @type {import('./foo').Foo}
+	 */
+	export let foo;
```

Comment:
These also seem unrelated, but possibly ok to keep because else prettier complains (?)